projects/uitdatabank/toc.json

```diff
@@ -17,23 +17,31 @@
     },
     {
       "type": "divider",
-      "title": "Guides"
+      "title": "Entry API"
     },
     {
-      "type": "group",
-      "title": "Entry API",
-      "items": [
-        {
-          "type": "item",
-          "title": "Requirements before going live",
-          "uri": "/docs/entry-api/requirements-before-going-live.md",
-          "slug": "entry-api/requirements-before-going-live"
-        }
-      ]
+      "type": "item",
+      "title": "API reference",
+      "uri": "reference/entry.json"
+    },
+    {
+      "type": "item",
+      "title": "Requirements before going live",
```

Comment:
I have deliberately not put this under a "Guides" group but kept this at the top level, since it's so important